packages/eslint-plugin/src/configs/vue.ts

```diff
@@ -10,11 +10,19 @@ const i18nDestructureRules = ["t", "tc", "te", "td", "d", "n"].map(
 export = {
   extends: [
     "plugin:vue/recommended",
+    "plugin:vue/vue3-recommended",
     "plugin:vuejs-accessibility/recommended",
     "plugin:@intlify/vue-i18n/recommended",
   ],
   plugins: ["vue", "vuejs-accessibility", "@intlify/vue-i18n"],
   rules: {
+    // Disable these rules after the Nuxt 3 migration
+    "vue/no-deprecated-dollar-listeners-api": "off",
+    "vue/no-v-for-template-key-on-child": "off",
+    "vue/no-deprecated-v-on-native-modifier": "off",
+    // TODO: disable after https://github.com/WordPress/openverse/issues/3268 is fixed
+    "vue/no-deprecated-props-default-this": "off",
```

Comment:
Nit: just for clarity, enable makes more sense. I'd expect "disabling" an ESLint rule to mean turning it off, but I guess here it means to disable the override. Maybe even "Remove these overrides..." instead of enable is clearer.

```suggestion
    // Enable these rules after the Nuxt 3 migration
    "vue/no-deprecated-dollar-listeners-api": "off",
    "vue/no-v-for-template-key-on-child": "off",
    "vue/no-deprecated-v-on-native-modifier": "off",
    // TODO: enable after https://github.com/WordPress/openverse/issues/3268 is fixed
    "vue/no-deprecated-props-default-this": "off",
```

Reply:
Definitely!